.github/workflows/manual-upgrade-testnet-l2.yml

```diff
@@ -109,11 +141,21 @@ jobs:
             az vm run-command invoke -g Testnet -n "${{needs.build.outputs.RESOURCE_STARTING_NAME}}-${{ matrix.host_id }}-${{ github.event.inputs.github_deploy_number }}"  \
             --command-id RunShellScript \
             --scripts '
-              cd /home/obscuro/go-obscuro/testnet/ \
-              docker compose down host enclave && \
-              docker compose pull && \
-              docker compose up host enclave
-            '
+              git clone --depth 1 -b ${{ steps.extract_branch.outputs.branch }} https://github.com/obscuronet/go-obscuro.git /home/obscuro/go-obscuro \
+            && cd /home/obscuro/go-obscuro/ \
+            && sudo go run /home/obscuro/go-obscuro/go/node/cmd  \
+              -is_genesis=${{ matrix.is_genesis }} \
+              -node_type=${{ matrix.node_type }} \
+              -is_sgx_enabled=true \
+              -host_id=${{ secrets[matrix.node_pk_addr] }} \
+              -l1_host=${{needs.build.outputs.L1_HOST}} \
```

Comment:
Is port needed here? The L2 deployment failure on the initial setup phase was failing because the default was different than what was deployed so I'm guessing this should also be wired?

Reply:
Ah interesting. It's not currently being set on the `manual-deploy-testnet-l2.yml` script so I think the default works for these nodes.

Here's a clipboard diff between the deploy and the upgrade scripts:
![Screenshot 2023-03-14 at 12 32 00](https://user-images.githubusercontent.com/98158711/225002451-8d33bc2f-e76c-48e1-ba78-3a32422924ee.png)

Reply:
Are the contract addresses injected from somewhere? As we most definitely would need them on an upgrade restart

Reply:
Yeah that's what the json thing is in this pr. We persist the contract addresses on the VM at deploy time and read them for restarts